tests/lib/assertions.py

```diff
@@ -72,17 +62,7 @@ def assert_201_response_messages(resp, environment):
         assert response.get("attributes").get("routingPlan") is not None
         assert response.get("attributes").get("routingPlan").get("id") != ""
         assert response.get("attributes").get("routingPlan").get("version") != ""
-
```

Comment:
Are we removing these on purpose? The URL in the response will be getting asserted as being incorrect with this change